### PR TITLE
Feature/203/connect offers page with backend

### LINKED ISSUE #203: (SP: 1) Connect offer page with backend

### PR DESCRIPTION
**Pull request by miller-113**
The pr introduce upgraded offerDetails card connected to backend.

### CODE REVIEW

**Review by Yaroslav-Yakovlev — APPROVED** [review]

**Review by mistercapxx — APPROVED** [review]
looks amazing! Best code I've ever seen!!!

**Review by CallMeMaverick — APPROVED** [review]

File: src/containers/offer-cards/OfferCards.tsx [hunk 1]
```diff
@@ -1,20 +1,46 @@
...
+  }, [])
+
+  const { response, loading } = useAxios<ItemsWithCount<Offer>, GetOfferParams>(
+    {
+      service: getOfferService,
+      defaultResponse: defaultResponses.itemsWithCount,
+      onResponseError
+    }
+  )
+  console.log(response.items)
```
Comment (nrevutska) [inline src/containers/offer-cards/OfferCards.tsx]: remove this console.log

File: src/pages/find-offers/FindOffers.tsx [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
+
 const FindOffers = () => {
   const [alignment, setAlignment] = React.useState<string>('left')
-  const items = ['Newest', 'Rating', 'Price low-high', 'Price high-low']
   return (
     <>
-      <SortMenu items={items} />
       <ToggleButtons alignment={alignment} setAlignment={setAlignment} />
-      <OfferDetails />
+      <OfferCards isSquare={alignment === 'left' ? false : true} />
```
Comment (nrevutska) [inline src/pages/find-offers/FindOffers.tsx]: remove the ternary operator here, I explained about it the other day.

File: src/pages/offer-details/OfferDetails.tsx [hunk 3]
```diff
@@ -14,39 +14,74 @@ import SpokenLanguages from '~/components/spoken-languages/SpokenLanguages'
...
+        />
         <Typography paragraph sx={styles.description} variant='body2'>
-          Hello. There are many variations of passages of Lorem Ipsum available,
-          but the majority have suffered alteration in some form, by injected
-          humour, or randomized words which don&apos;t look even slightly
-          believable...
+          {/* there must be author comment */}
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Culpa minus
+          est nisi ipsa nihil quos, a placeat voluptas at quae voluptate eius
+          dolorem sapiente voluptatibus optio numquam cumque molestiae possimus!
```
Comment (nrevutska) [inline src/pages/offer-details/OfferDetails.tsx:67]: why do we still use this stub text here, and not the real comment taken from the backend?

**Review by nrevutska — CHANGES_REQUESTED** [review]

File: src/pages/offer-details/OfferDetails.tsx (hunk 3 above)
Comment (miller-113) [reply src/pages/offer-details/OfferDetails.tsx:67]: Only title is implemented in the backend, description has not been implemented yet.

**Pull request merged**